flusher_test.go

```diff
@@ -174,7 +175,7 @@ func testFlushTraceDatadog(t *testing.T, protobuf, jsn io.Reader) {
 	assert.NoError(t, err)
 
 	server.HandleTracePacket(packet)
-	server.Flush()
+	server.Flush(context.TODO())
```

Comment:
I think we actually want `context.Background()` for all of these, no?

Reply:
Ha! Yes, I was doing TODO out of habit for a temporary fix when in fact `Background` was the permanent answer here. :) Thanks!